api/envoy/service/auth/v3/attribute_context.proto

```diff
@@ -176,4 +182,7 @@ message AttributeContext {
 
   // Dynamic metadata associated with the request.
   config.core.v3.Metadata metadata_context = 11;
+
+  // TLS session details of the underlying connection.
+  TLSSession tls_session = 12;
```

Comment:
Can you document that this is not enabled by default and needs configuration to turn on? And then ref link to the other configuration?

Reply:
Added.